### PR TITLE
ADaaS revision

### PR DESCRIPTION
**Pull request by bc-devrev**
https://app.devrev.ai/devrev/works/ISS-128134

### CODE REVIEW

File: fern/docs/pages/adaas/overview.mdx [hunk 1]
```diff
@@ -5,7 +5,44 @@ You can read more about Airdrop in the [general documentation](https://docs.devr
...
+	worker <-- Upload/download artifacts --> s3
+	worker <-- ADaaS messages and REST API --> airdrop
+
+	
+	subgraph DevRev
+	s3interact <-- Prepare upload/download URL --> s3
+	airdrop <-- Download/upload artifacts --> s3
+	end
+
+```
```
Comment (erazemk) [inline fern/docs/pages/adaas/overview.mdx:45]: The default color scheme seems non-pleasant, we need to find a better color scheme.
image

**Review by erazemk — COMMENTED** [review]
Nit about the Mermaid Markdown color scheme, besides that LGTM.

File: fern/docs/pages/adaas/overview.mdx (hunk 1 above)
Comment (erazemk) [reply fern/docs/pages/adaas/overview.mdx:45]: I just now saw you added an init function, why? Was the default color scheme not good enough?

Comment (bc-devrev) [reply fern/docs/pages/adaas/overview.mdx:45]: This color scheme uses DevRev colors as specified in the [Brand Guidelines](https://www.figma.com/design/U7Cj661r6a38Z87L1hURM8/Brand-Guidelines?node-id=3-149&t=wa1zOrRQo9BO93uF-1). If you have a suggestion for a better way to apply those colors I'm open to it.

Comment (erazemk) [reply fern/docs/pages/adaas/overview.mdx:45]: I don't see these exact colors in the Figma link you shared, but also, I think having the diagram nicely fit is probably preferrable. If needed, we can ask one of the designers to choose a color scheme that would fit on this specific page.

Comment (bc-devrev) [reply fern/docs/pages/adaas/overview.mdx]: You are correct about the colors. I now recall that I used the colors from some other diagram which, although it came from design, apparently didn't conform to the brand guidelines. I don't have time to mess around with it anymore so I have removed the color settings and just kept the font.

Comment (erazemk) [reply fern/docs/pages/adaas/overview.mdx]: Sure, IMO the default looks the best anyway. Thanks for the changes!

**Review by erazemk — APPROVED** [review]

File: fern/docs/pages/adaas/overview.mdx (hunk 1 above)
Comment (erazemk) [reply fern/docs/pages/adaas/overview.mdx]: Ah, my review doesn't count anyway 😂

**Review by Atul-Butola — APPROVED** [review]

**Comment by bc-devrev** [comment]
@radovan-jorgic @samod let me know if you have any feedback; otherwise I will merge this change tomorrow.

**Pull request merged**